src/tcgwars/logic/impl/gen7/UnbrokenBonds.groovy

```diff
@@ -3580,7 +3580,7 @@ public enum UnbrokenBonds implements LogicCardInfo {
             energyCost C, C
             onAttack {
               damage 70
-              flip {afterDamage{discardDefendingEnergy();discardDefendingEnergy()}}
+              flip { afterDamage{ 2.times{ discardDefendingEnergy() } } }
```

Comment:
Interesting Groovy feature. Maybe this could be used for the instances of taking two prize cards you've somewhat touched already too?

Reply:
Could do indeed! Wanna confirm it working well but if it does will def update those 👍